app/blueprints/self_serve/routes.py

```diff
@@ -147,22 +136,42 @@ def build_section():
                 "hover_info": {"title": f["builder_display_name"], "pages": f["pages"]},
             }
         )
-    return render_template("build_section.html", available_forms=available_forms, form=form)
-
+        # save to db here
+    return render_template("create_section.html", available_forms=available_forms, form=form)
 
-@self_serve_bp.route("/add_question", methods=["GET", "POST"])
-def add_question():
-    form = QuestionForm()
-    question = form.as_dict()
+@self_serve_bp.route("/create_form", methods=["GET", "POST"])
+def create_form():
+    form = FormForm()
     if form.validate_on_submit():
-        save_question(question)
-        flash(message=f"Question '{question['title']}' was saved")
+        new_form = {
```

Comment:
for the overall process shall we add more logs to determine the process is working fine ?

Reply:
ill leave this for now, as its Sarahs temporary solution (until we have more info on the frontend parts)